contrib/cirrus/packer/ubuntu_packaging.sh

```diff
@@ -122,6 +122,10 @@ INSTALL_PACKAGES=(\
     zip
     zlib1g-dev
 )
+DOWNLOAD_PACKAGES=(\
+    cri-o-1.17_1.17.4
```

Comment:
is there the corresponding package bump in fedora?

Reply:
Sadly, this is a really F'd up aspect of Ubuntu-land and our cri-o packages.  The version number is part of the package name (they do the same thing with the damn kernel).  I would MUCH prefer to just specify `crio`, where (as in Fedora-land) it simply picks up the latest available distro version and dependencies.

That said, since we're just downloading the package, and will install it at runtime, I can be flexible here if there is another/different solution needed.  Also remember, while conmon will share images produced from this project, it does _NOT_ have to always be using the exact same images as libpod.  In other words, each project can move to new (or old) images as needed by that project.